gabc/details.html

```diff
@@ -244,7 +248,7 @@ <h3 id="translation">Translation and text above notes</h3>
 <p>It is also possible to add text above notes in two ways: the first is to set the text at the syllable text level between <code>&lt;alt&gt;</code> markups (alt like <em>above lines text</em>). This text will be left-aligned with the first note of the syllable. It is also possible to typeset text entering <code>[alt:my text]</code> in the notes, which will align "my text" with the note following.</p>
```

Comment:
- `<alt> markups` should probably be `<alt> markup tags`
- `(alt like above lines text)` should probably be `(alt, short for above lines text)`
- `the note following` should probably be `the next note`.